app/models/conversion_host.rb

```diff
@@ -158,14 +158,91 @@ def ipaddress(family = 'ipv4')
   # @raise [MiqException::MiqSshUtilHostKeyMismatch] if conversion host key has changed
   # @raise [JSON::GeneratorError] if limits hash can't be converted to JSON
   # @raise [StandardError] if any other problem happens
-  def apply_task_limits(path, limits = {})
-    connect_ssh { |ssu| ssu.put_file(path, limits.to_json) }
+  def apply_task_limits(task_id, limits = {})
+    connect_ssh do |ssu|
+      ssu.put_file("/tmp/#{task_id}-limits.json", limits.to_json)
+      ssu.shell_exec("mv /tmp/#{task_id}-limits.json /var/lib/uci/#{task_id}/limits.json", nil, nil, nil)
```

Comment:
`task_id` comes from [`migration_task.id`](https://github.com/ManageIQ/manageiq/pull/19698/files#diff-8dffd85ef5e1772dce26180aa8ff6d65R100) so reduced risk of shell escape attacks here, awesome spawn would still be preferred though

Reply:
Using AwesomeSpawn now.